bazaarvoice/emodb
# postman tests for add role endpoint

## What Are We Doing Here?

New Postman tests have been added for uac/1/role/{group}/{id} emodb endpoint
## How to Test and Verify

1. Run this PR on postman Jenkins Pipeline 
2. Verify that all tests are passed and that tests are run for uac/1/role/{group}/{id} endpoint

## Risk
no changes to existing code. new unstable tests can be added to test suite run
### Level 

`Low`

### Required Testing

`Manual`

### Risk Summary

No risks are expected for existing functionality, however new unstable tests can be introduced to our test suite

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
